Makefile

```diff
@@ -123,50 +119,44 @@ test/unit: ## Run the unit tests
 	$(Q)go test -count=1 -short ./pkg/...
 	$(Q)go test -count=1 -short ./internal/...
 
-test/subcommand: test/subcommand/test-local test/subcommand/scorecard test/subcommand/alpha-olm
+# CI tests.
+.PHONY: test/ci
+
+test/ci: test/markdown test/sanity test/unit install test/subcommand test/e2e ## Run the CI test suite
 
-test/subcommand2:
-	./ci/tests/subcommand.sh
+# Subcommand tests.
+.PHONY: test/subcommand test/subcommand/test-local test/subcommand/scorecard test/subcommand/olm-install
+
+test/subcommand: test/subcommand/test-local test/subcommand/scorecard test/subcommand/olm-install
 
 test/subcommand/test-local:
-	./hack/tests/test-subcommand.sh
+	./hack/tests/subcommand.sh
 
 test/subcommand/scorecard:
-	./hack/tests/scorecard-subcommand.sh
+	./hack/tests/subcommand-scorecard.sh
 
-test/subcommand/scorecard2:
-	./ci/tests/scorecard-subcommand.sh
+test/subcommand/olm-install:
+	./hack/tests/subcommand-olm-install.sh
 
-test/subcommand/alpha-olm:
-	./hack/tests/alpha-olm-subcommands.sh
+# E2E tests.
+.PHONY: test/e2e test/e2e/go test/e2e/ansible test/e2e/ansible-molecule test/e2e/helm
 
 test/e2e: test/e2e/go test/e2e/ansible test/e2e/ansible-molecule test/e2e/helm ## Run the e2e tests
 
 test/e2e/go:
 	./hack/tests/e2e-go.sh $(ARGS)
 
-test/e2e/go2:
```

Comment:
I believe these were added for `openshift/release`, because we only need to run the tests without the image builds or something like that, will this be able to pass the prow tests?

Reply:
Never mind, I saw the new Makefile below, which I assuming is what we'll be running in prow. That is cleaner than just duplicating the tasks, good idea :+1: